single_line_class.rb

```diff
@@ -0,0 +1,27 @@
+#Prefer a single-line format for class definitions with no body
+class FooError < StandardError
+end
+
+class Test
+	# Avoid single-line methods.
+    def too_much; puts 'pravin'; puts 'mishra' end 
+    
+    # bad
+    e = M * c ** 2
```

Comment:
__[rubocop]__
* Useless assignment to variable - `e`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Space around operator `**` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)